homeassistant/components/lock/strings.json

```diff
@@ -8,11 +8,13 @@
     },
     "condition_type": {
       "is_locked": "{entity_name} is locked",
-      "is_unlocked": "{entity_name} is unlocked"
+      "is_unlocked": "{entity_name} is unlocked",
+      "is_open": "{entity_name} is unlocked"
```

Comment:
This is technically correct as open means unlocked, but it would be easier for the user in the frontend if this would be"{entity_name} is open".